CHANGELOG

```diff
@@ -6,6 +6,7 @@ version 3.3 (milestone)
 * fedora_updates module: add check_security option, by @jakubjedelsky
 * github module: open github home page if no repo is configured and you're gettin all notifications, by @rixx
 * volume_status module: add pulseaudio support, by @guiniol
+* spaceapi module: go to hackerspace website onclick, by @rixx
```

Comment:
Generally we haven't added to the change log in PRs I think it may also lead to lots of unnecessary merge conflicts if we start doing this.

@ultrabug what is your take on this